ocrd_eval.sample.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "@id": "https://github.com/OCR-D/quiver/tree/data/evaluations/wf1-data345-eval1.json",
+    "label": "OCR workflow 1 on workspace 345",
+    "metadata": {
+      "ocr_workflow": {
+        "@id": "https://github.com/OCR-D/quiver/tree/data/workflows/1.nf",
+        "label": "OCR Workflow 1"
+      },
+      "eval_workflow": {
+        "@id": "https://github.com/OCR-D/quiver/tree/data/workflows/eval1.nf",
+        "label": "Evaluation Workflow 1"
+      },
+      "gt_workspace": {
+        "@id": "https://gt.ocr-d.de/workspace/789",
+        "label": "GT workspace 789 (19th century fraktur)"
+      },
+      "ocr_workspace": {
+        "@id": "https://github.com/OCR-D/quiver/tree/data/workspaces/3000.ocrd.zip",
+        "label": "OCR result workspace 3000"
+      },
+      "eval_workspace": {
+        "@id": "https://github.com/OCR-D/quiver/tree/data/workspaces/345.ocrd.zip",
+        "label": "Evaluation Workspace 345"
+      },
+      "workflow_steps": {
+        "0": "Processor A",
+        "1": "Processor B"
+      },
+      "workflow_model": "Fraktur_GT4HistOCR",
+      "document_metadata": {
+        "fonts": [
+          "antiqua",
+          "fraktur"
+        ],
+        "publication_century": "1800-1900",
+        "publication_decade": "1850-1860",
+        "publication_year": 1855,
```

Comment:
Currently, Matthias' [GT metadata tool](https://tboenig.github.io/gt-metadata/document-your-gt.html) allows for providing an upper and lower bound for the publication date which I translate to `publication_century`. Do we need a more granular date in QuiVer (or any other tool we might use this JSON schema for)? If not, I suggest to settle with `publication_century` for now. 

What do you think, @kba?

Reply:
I think `publication_year` would be enough, decade and century are derived from it.

Reply:
Alright :)